frontend/views/components/Chatroom/Message.vue

```diff
@@ -1,13 +1,13 @@
-<template lang="pug">
-.c-message(:class="[variant, isSameSender && 'sameSender']")
-  avatar.c-avatar(:src='avatar' :class="{ 'alignToText': !hasWhoInvisible }" aria-hidden='true')
+<template lang='pug'>
+.c-message(:class='[variant, isSameSender && "sameSender"]')
+  avatar.c-avatar(:src='avatar' :class='{ alignToTex: !hasWhoInvisible }' aria-hidden='true')
```

Comment:
Typo here (should be `alignToText`). I think I agree, that it really would be great if our tools noticed all such problems, but that luxury does come at a cost: it makes us less vigilant ourselves, and when our tools aren't ready to take over our jobs, it can lead to disaster.

Random search for "yoda emoji" reveals: 
![Screen Shot 2019-09-09 at 6 20 04 PM](https://user-images.githubusercontent.com/138706/64570398-83850500-d32e-11e9-9250-cb7c1c1f88d4.jpg)

Reply:
wise words. fixed!